relay/api/resources.py

```diff
@@ -213,7 +213,9 @@ def get(self, args, user_address: str):
                 events.append(gevent.spawn(proxy.get_all_network_events,
                                            user_address=user_address,
                                            from_block=from_block))
-        gevent.joinall(events, timeout=5)
+        gevent.joinall(events, timeout=10)
+        if events is None:
```

Comment:
I am not convinced that this will fix the error. Events is a list and can not be None